dashboard/bin/scheduled_ops_emails

```diff
@@ -6,24 +6,38 @@ require_relative '../config/environment'
 def main
   Dir.chdir(dashboard_dir) do
     # Get all workshops that start two weeks or 3 days from today
-    workshops = DASHBOARD_DB.fetch('
+    upcoming_workshops = DASHBOARD_DB.fetch('
```

Comment:
I think a good approach for encapsulating here would be to define a class method on Workshop so you could call `Workshop.upcoming` or `Workshop.ending`

As a bonus you'd be able to unit test those through the rails testing setup.